S/sequentialknn/build_tarballs.jl

```diff
@@ -0,0 +1,67 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, BinaryBuilderBase
+
+name = "sequentialknn"
+version = v"0.1"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource(
+        "https://github.com/cgeoga/sequentialknn",
+        "ef59da866a14ff5a53e27295cfd6f8d9ca2999c3"),
+]
+
+# Bash recipe for building across all platforms.
+#
+# CG: this is kind of gross looking, but I just had a lot of issues with the
+# ${libext} and stuff not working. And on windows, rust doesn't put lib in front
+# of the library name. So there is some manual tinkering here just to make all
+# the file names exist as expected.
+script = raw"""
+cd $WORKSPACE/srcdir/sequentialknn/
+export LIBRARY_PATH="${libdir}:${LIBRARY_PATH}"
+export DYLD_FALLBACK_LIBRARY_PATH="${libdir}"
+export CPATH="${includedir}:${CPATH}"
+cargo build --release
+echo "TARGET = ${target}"
+case "${target}" in
+    *apple-darwin*)
+        ext=dylib
+        ;;
+    *linux*)
+        ext=so
+        ;;
+    *w64*)
+        ext=dll
+        ;;
+esac
```

Comment:
[`dlext`](https://docs.binarybuilder.org/stable/build_tips/#Automatic-environment-variables) does this already.
```suggestion
```